src/monstergenerator.cpp

```diff
@@ -907,6 +925,14 @@ void mtype::load( const JsonObject &jo, const std::string &src )
     optional( jo, was_loaded, "fungalize_into", fungalize_into, string_id_reader<::mtype> {},
               mtype_id() );
 
+    if( jo.has_array( "attack_effs" ) ) {
+        for( const JsonObject effect_jo : jo.get_array( "attack_effs" ) ) {
```

Comment:
```suggestion
    if( jo.has_array( "attack_effs" ) ) {
        atk_effs.clear();
        for( const JsonObject effect_jo : jo.get_array( "attack_effs" ) ) {
```
You'll need to clear this vector if `"attack_effs"` is defined, or else they'll just accumulate in inherited objects that are supposedly overwriting it